chart/compass/charts/connector/templates/service-role-binding.yaml

```diff
@@ -11,27 +11,22 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 spec:
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+      release: {{ .Release.Name }}
+  action: ALLOW
   rules:
-    - services: ["{{ template "fullname" . }}-internal.{{ .Release.Namespace }}.svc.cluster.local"]
-      paths: ["*"]
-      methods: ["*"]
----
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
-metadata:
-  name: {{ template "fullname" . }}-internal
-  namespace: {{ .Release.Namespace }}
-  labels:
-    release: {{ .Release.Name }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    app.kubernetes.io/name: {{ template "name" . }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-spec:
-  subjects:
-{{ tpl (toYaml .Values.istio.rbac.connectorInternalApi.subjects | indent 4) . }}
-  roleRef:
-    kind: ServiceRole
-    name: {{ template "fullname" . }}-internal
----
+    - from:
+      - source:
+          principals:
+          - {{ tpl .Values.istio.rbac.connectorInternalApi.compassDirector . }}
+          - {{ tpl .Values.istio.rbac.connectorInternalApi.compassConnectorTests . }}
+    - to:
+      - operation:
+          ports:
+          - "{{ .Values.global.connector.graphql.internal.port }}"
```

Comment:
does it make sense instead of pointing to ports to simply say  methods: ["GET", "POST"] or  paths: ["/*"] or not specifying anything in "to" (in the docs it says if nothing is specified in to, it will match all paths, methods,ports, etc for this workload) or is there any benefit for us to use ports? (This is more a question, out of curiosity)

Reply:
We have to keep it, because we want to ensure that the access to external ports is allowed for anyone, as described [here](https://github.com/kyma-incubator/compass/pull/1536/files#r495713387)